README.md

```diff
@@ -39,39 +39,19 @@ The fields in the table below can be used in these parts of STAC documents:
 | view:moon_azimuth    | number | Moon azimuth angle. From the scene center point on the ground, this is the angle between truth north and the moon. Measured clockwise in degrees (0-360). |
 | view:moon_elevation  | number | Moon elevation angle. The angle from the tangent of the scene center point to the moon. Measured from the horizon in degrees (`0`-`90`). |
 
-The angles `off_nadir`, `incidence_angle`, and `sun_elevation` / `moon_elevation` are angles measured on a 2d plane formed:
-sensor location, sub-sensor point on the earth, the sun / moon, and the center of the viewed area.
-
-The off-nadir angle and the incidence angle are related. When the off-nadir angle is low (low incidence angle) then the 
-two angles are approximately equal. However, at high off-nadir angles with high altitude sensors the curvature of the earth
-has an impact and the two angles are no longer equivalent.
-
-The angles `azimuth` and `sun_azimuth` / `moon_azimuth` indicate the position of the viewed scene and
-the sun / moon by the angle from true north, as shown below.
-
-Example:
-
-```js
-{
-  "stac_version": "1.1.0",
-  "stac_extensions": [
-    "https://stac-extensions.github.io/view/v1.0.0/schema.json"
-  ],
-  "id": "20171110",
-  "type": "Feature",
-  ...
-  "properties": {
-    "platform": "mysatellite",
-    "instruments": ["mycamera1", "mycamera2"],
-    "constellation": "allmysatellites",
-    "view:off_nadir": 0,
-    "view:incidence_angle": 90,
-    "view:azimuth": 23.9,
-    "view:sun_elevation": 45.0,
-    "view:sun_azimuth": 56.4
-  }
-}
-```
+The angles `off_nadir` and are angles measured on a 2d plane formed by the sensor location, the sub-satellite point on
+the earth, and the center of the target area as shown in the diagram below.  Grazing angle is shown as it is frequently
+used but it is not included in this extension because it is simply the complement of the incidence angle. When the off-nadir
+angle is low (low incidence angle) then the two angles are approximately equal. However, at high off-nadir angles with high
+altitude sensors the curvature of the earth has an impact and the two angles are no longer equivalent.
+
+![View angle diagram](images/view-angles-1.png)
+
+The `azimuth` (`sun_azimuth`, `moon_azimuth`), and the `elevation` (`sun_elevation`, `moon_elevation`) angles are measured 
+as per the diagram below. Azimuth angles are measured as degrees from North, and elevation angles are measured up from 
+the target plane to the body (satellite, sun, or moon).
+
+![View angle diagram](images/view-angles-2.png)
```

Comment:
I read the definition of `azimuth` in this new diagram as **not** the same as what was stated in https://github.com/stac-extensions/view/issues/2#issuecomment-1163372255. I read this new diagram as indicating that `view:azimuth` is measured at the target point P not at the sub satellite point S, whereas that older comment indicates that `view:azimuth` is measured from S.

The left and right reference images for that discussion are here: https://github.com/stac-extensions/view/issues/2#issuecomment-1162357118. 

Is my understanding of this PR incorrect? Was @matthewhanson original comment stating that `view:azimuth` represented the left side picture incorrect? Is this new diagram in this PR incorrect? Something else?

Reply:
@CloudNiner I think I've been wrong about every time on this thread, so is my diagram now wrong or my previous comment wrong?  Maybe both?

I looked up other diagrams and came to the conclusion that azimuth should be measured from the observer, or target point back to the celestial body or satellite.  Yes, that's the opposite of what I told you in https://github.com/stac-extensions/view/issues/2#issuecomment-1162357118 

But I could also see a viewing azimuth and viewing elevation as measured from the satellite, but that makes less sense for the sun or moon.